vendor/assets/javascripts/github_buttons.js

```diff
@@ -12,177 +12,89 @@
  * limitations under the License.
  *
  * Source: https://github.com/mdo/github-buttons/blob/7c1da76484288ce76fa061362fc1c1f0db1f6553/src/js.js
- * Modification: Changed params to read attributes from data-params
+ * Modification: Changed params to read attributes from data attributes
  *               Execute only when .github-btn exists
  *               Remove title update (mdo/github-buttons@cbf5395b)
+ *               Stripped to minimal needed code with no dependencies (show amount of stars)
  */
 
-if ($(".github-btn").length) {
+if (document.querySelectorAll('.github-btn').length) {
   (function() {
     'use strict';
 
-  // Read a page's GET URL variables and return them as an associative array.
-  // Source: https://jquery-howto.blogspot.com/2009/09/get-url-parameters-values-with-jquery.html
-  function getUrlParameters() {
-    var vars = [];
-    var hash;
-    var hashes = $('.github-btn').attr('data-params').split('&');
-
-    for (var i = 0; i < hashes.length; i++) {
-      hash = hashes[i].split('=');
-      vars.push(hash[0]);
-      vars[hash[0]] = hash[1];
+    // Add commas to numbers
+    function addCommas(n) {
+      return String(n).replace(/(\d)(?=(\d{3})+$)/g, '$1,');
     }
 
-    return vars;
-  }
-
-  // Add commas to numbers
-  function addCommas(n) {
-    return String(n).replace(/(\d)(?=(\d{3})+$)/g, '$1,');
-  }
-
-  function jsonp(path) {
-    var xhr = new XMLHttpRequest();
-    xhr.open('GET', path, true);
-    xhr.responseType = 'json';
-    xhr.onload = function() {
-      var status = xhr.status;
-      if (status === 200) {
-        var res = { data: { stargazers_count: xhr.response.stargazers_count } };
-        callback(res);
-      } else {
-        console.log("Request to github failed with status:", status)
-      }
-    };
-    xhr.send();
-  }
-
-  var parameters = getUrlParameters();
-
-  // Parameters
-  var user = parameters.user;
-  var repo = parameters.repo;
-  var type = parameters.type;
-  var count = parameters.count;
-  var size = parameters.size;
-  var v = parameters.v;
-
-  // Elements
-  var button = document.querySelector('.gh-btn');
-  var mainButton = document.querySelector('.github-btn');
-  var text = document.querySelector('.gh-text');
-  var counter = document.querySelector('.gh-count');
-
-  // Constants
-  var LABEL_SUFFIX = ' on GitHub';
-  var GITHUB_URL = 'https://github.com/';
-  var API_URL = 'https://api.github.com/';
-  var REPO_URL = GITHUB_URL + user + '/' + repo;
-  var USER_REPO = user + '/' + repo;
-
-  window.callback = function(obj) {
-    if (obj.data.message === 'Not Found') {
-      return;
-    }
-
-    switch (type) {
-      case 'watch':
-        if (v === '2') {
-          counter.textContent = obj.data.subscribers_count && addCommas(obj.data.subscribers_count);
-          counter.setAttribute('aria-label', counter.textContent + ' watchers' + LABEL_SUFFIX);
+    function jsonp(path) {
+      var xhr = new XMLHttpRequest();
```

Comment:
Not saying this needs to be solved right now in the PR, but I would be very keen to get rubygems.org away from `XMLHttpRequest`. [fetch](https://caniuse.com/fetch) is now covered by the vast majority of browsers and there's plenty of libraries that put it behind a nice abstraction like https://github.com/sindresorhus/ky